FizzBuzzApp/Program.cs

```diff
@@ -1,3 +1,15 @@
 ﻿// See https://aka.ms/new-console-template for more information
-
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+for (int i = 1; i < 101; i++) {
+    if (i % 3 == 0 && i % 5 == 0) {

```

Comment:
This is fine for a first version, but as things get more and more complex with extra rules etc, you may want to think about how not to have to explicitly define each rule combination.
If you were to add a new rule (e.g. Bang) here you'd need to refactor or add lots of checks for every combination of Fizz/Buzz/Bang

Reply:
Sorry yes, I had already changed it in Part 2 before I saw this. Let me know if you have any comments on that structure when you get to it!